src/color/getColorContrastScores.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import getColorContrast from './getColorContrast'
+
+import type { ContrastScores } from '../types/color'
+
+/**
+ * Returns the contrast scores between two colors based on the
+ * [contrast scores recommended by W3](https://www.w3.org/TR/WCAG20/#visual-audio-contrast).
+ *
+ * @example
+ * const scores = getColorContrastScores('#444', '#fff');
+ */
+export default function getColorContrastScores(
```

Comment:
Wondering if this something like `meetsContrastGuidelines` might be better since we really aren't returning scores as much as checking if they comply with the W3s guidelines for different scenarios.

Reply:
Yeah, I like that name way better! 😄